plutus-book/doc/01-preface.adoc

```diff
@@ -19,7 +19,7 @@ and understand the basics of Haskell programming prior to reading this book.
 Come and explore the world of Plutus and learn our new language through
 real life scenarios and sample code snippets!
 
-*Disclaimer:*
+##Disclaimer:##
```

Comment:
Sections use `=` not `#`, that's markdown :) Also, you probably don't want a trailing comma if it's a heading.